onnx/__init__.py

```diff
@@ -18,31 +18,31 @@
 
 import google.protobuf.message
 
-from typing import Union, Text, IO, Optional, cast, TypeVar, Any
+from typing import Union, IO, Optional, cast, TypeVar, Any
 
 
 # f should be either readable or a file path
-def _load_bytes(f: Union[IO[bytes], Text]) -> bytes:
+def _load_bytes(f: Union[IO[bytes], str]) -> bytes:
     if hasattr(f, 'read') and callable(cast(IO[bytes], f).read):
         s = cast(IO[bytes], f).read()
     else:
-        with open(cast(Text, f), 'rb') as readable:
+        with open(cast(str, f), 'rb') as readable:
             s = readable.read()
     return s
 
 
 # str should be bytes,
 # f should be either writable or a file path
-def _save_bytes(str: bytes, f: Union[IO[bytes], Text]) -> None:
+def _save_bytes(str: bytes, f: Union[IO[bytes], str]) -> None:
     if hasattr(f, 'write') and callable(cast(IO[bytes], f).write):
         cast(IO[bytes], f).write(str)
     else:
-        with open(cast(Text, f), 'wb') as writable:
+        with open(cast(str, f), 'wb') as writable:
```

Comment:
Linux-CI failed because mypy typecheck error here: https://dev.azure.com/onnx-pipelines/onnx/_build/results?buildId=24804&view=logs&j=26ad7b42-9ea7-5ad4-891d-e581c481314e&t=36dc6483-53ad-5f31-0c8e-c9104543cd82

Reply:
Looks like the function param is redefining `str`. I can try changing it. Is there a fix you would recommend?

Reply:
I don't have a strong preference about the solution as long as mypy typecheck is happy. It's interesting that only L40 complains this error. (I saw similar a change at L29). Actually `with open(cast(str, f), 'wb') as writable:` looks fine to me so it is possible that it is a false alarm? Or, according to the error message, you might try to remove the cast since it says it is unnecessary. (although I think it should be needed)

Reply:
Renamed the first param to `content`